flickfinder-backend/src/main/java/com/flickfinder/flickfinderbackend/controllers/MovieController.java

```diff
@@ -43,15 +43,19 @@ public Flux<Movie> getTrendingMovies() {
   
 //    @GetMapping("/movies")
 //    public List<Movie> getMovies() {
-//        return;
-//        // Return a list of available movies
-//    }
+//        return movieService.getMovies();
 //
 //
-//    @PostMapping("/recommendations")
-//    public List<Movie> getRecommendations(@RequestBody Movie movie) {
-//        // Implement recommendation algorithm and return recommended movies
 //    }
+
+
+//    @PostMapping("/recommendations")
```

Comment:
This might be part of your next issue to pull a movie id from the user's saved movies rather than hard coding, so not necessary for this issue. You'll need post mapping to be uncommented and you might be returning just the id (int) rather than a list of movies.

But that can be worked out with your next issue!